ngraph/core/src/op/experimental_detectron_generate_proposals.cpp

```diff
@@ -0,0 +1,67 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/experimental_detectron_generate_proposals.hpp"
+#include "itt.hpp"
+#include "ngraph/attribute_visitor.hpp"
+#include "ngraph/op/util/op_types.hpp"
+#include "ngraph/validation_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(op::v6::ExperimentalDetectronGenerateProposalsSingleImage,
+                       "ExperimentalDetectronGenerateProposalsSingleImage",
+                       6);
+
+op::v6::ExperimentalDetectronGenerateProposalsSingleImage::
+    ExperimentalDetectronGenerateProposalsSingleImage(const Output<Node>& im_info,
+                                                      const Output<Node>& anchors,
+                                                      const Output<Node>& deltas,
+                                                      const Output<Node>& scores,
+                                                      const Attributes& attrs)
+    : Op({im_info, anchors, deltas, scores})
+    , m_attrs(attrs)
+{
+    constructor_validate_and_infer_types();
+}
+
+shared_ptr<Node> op::v6::ExperimentalDetectronGenerateProposalsSingleImage::clone_with_new_inputs(
+    const OutputVector& new_args) const
+{
+    check_new_args_count(this, new_args);
```

Comment:
Please, add `NGRAPH_OP_SCOPE` macros like in the https://github.com/openvinotoolkit/openvino/pull/3730

Reply:
Done.